examples/classification/readme.md

```diff
@@ -0,0 +1,73 @@
+# HierarQML
+
+This tutorial guides on using HierarQcal to create quantum circuit models for classification tasks. It uses the modern stack of Pytorch Lightning and PennyLane for classical and quantum machine learning.
+
+## Setup
+
+Create a Python environment and install the required packages using
+
+```
+pip install -r requirements.text
+```
+
+Set the Python Path using `export PYTHONPATH=.` before executing any commands later on, or you will get errors like ModuleNotFoundError: No module named ''.
+
+## Project Structure
+
+To go through the main project structure, let's train a QCNN on GTZAN data.
+
+### Directory Structure
+
+```
+├── hierarqml
```

Comment:
In this directory can you add a .gitignore file, that removes things like .venv and the other default python stuff. Then also mention in the readme that someone can copy the whole folder out and turn it into their own separate git project